### PR TITLE
Feature/ios offline maps

### PR DESCRIPTION
**Pull request by j8seangel**
This PR includes the localTiles component for iOS.

Is not working yet because the image doesn't appear on the map and I can't understand why here: https://github.com/Vizzuality/forest-watcher/compare/feature/ios-offline-maps?expand=1#diff-44b2bfcf63bf79541d50fced09f7b80eR35

But at least we have the component available to remove [this](https://github.com/Vizzuality/forest-watcher/compare/feature/ios-offline-maps?expand=1#diff-99b815461a9cdd3c23d4417085c8b531L621)

### CODE REVIEW

**Comment by sorodrigo** [comment]
Should we merge now even though it isn't complete? Or should we wait?

**Comment by sorodrigo** [comment]
Could you review my changes @j8seangel

File: app/components/map/index.js [hunk 1]
```diff
@@ -315,13 +314,6 @@ class Map extends Component {
     });
   }
 
-  getMapKey = () => {
```
Comment (j8seangel) [inline app/components/map/index.js]: Awesome catch! we got crazy with this and was much simpler 👏

File: ios/AirGoogleMaps/AIRGoogleMap.m [hunk 2]
```diff
@@ -95,6 +95,7 @@ - (void)insertReactSubview:(id<RCTComponent>)subview atIndex:(NSInteger)atIndex
   } else if ([subview isKindOfClass:[AIRGoogleMapLocalTile class]]) {
     AIRGoogleMapLocalTile *tile = (AIRGoogleMapLocalTile*)subview;
     tile.tileLayer.map = self;
+    tile.tileLayer.opacity = 1;
```
Comment (j8seangel) [inline ios/AirGoogleMaps/AIRGoogleMap.m]: 👏 💃 🎉

**Comment by j8seangel** [comment]
Remember to run `npm install` as I've just merged the library changes.

**Pull request merged**